lighty-examples/lighty-cluster-app/src/main/java/io/lighty/examples/controllers/cluster/Main.java

```diff
@@ -136,11 +138,11 @@ public void start(String[] args, boolean registerShutdownHook) {
 
     private void startLighty(ControllerConfiguration controllerConfiguration,
                              RestConfConfiguration restConfConfiguration,
-                             NetconfConfiguration netconfSBPConfiguration, boolean registerShutdownHook,
-                             boolean k8sDeployment)
+                             NetconfConfiguration netconfSBPConfiguration, boolean registerShutdownHook)
             throws ConfigurationException, ExecutionException, InterruptedException {
 
         //1. initialize and start Lighty controller (MD-SAL, Controller, YangTools, Akka)
+        LOG.info("Trying to start LightyController ...");
```

Comment:
Change this message to something like "Starting LightyController...". This sounds too pesimistic :).